templates/ci/android.yml.metadata

```diff
@@ -0,0 +1,3 @@
+{
+  "iconName": "Gradle"
```

Comment:
Any reason we're not using an Android icon? (like this one: https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/AndroidSigningV3/icon.png)

Reply:
That's an Android signing one (note the badge it is wearing) but I agree

Reply:
Hrmm... I didn't notice that.  I suppose that's 'a reason'.  (It would still be nice, though. :) )

Reply:
The Gradle icon an artifact from my first attempt to associate the primary task icon with the template; The template uses the `Gradle` task. But I agree than an Android logo makes more sense.

Reply:
Fixed. Thanks David for the Android icon.